### DISCUSSION TITLE
Release Version updates how often?

### DISCUSSION THREAD

**Discussion by LTSpenny** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2702] [Q&A] [answered]
Hello, just wondering if anyone knows about how often the release version gets updated? 

Thanks for info.

**Comment by radek8** [comment, marked as answer]
At random
Whenever there is something to improve and bigtreetech approves and merges it.

**Comment by burneme** [comment]
The next regarding "**versions**": Is there a place where old versions are still available?

**Comment by radek8** [comment]
Yes.
In principle, github is made in such a way that it can be returned upstream to any point.

**Reply by burneme** [reply]
Thanks, helped me a lot. Meanwhile I found the old releases. My problem is, that at a undefined time the print stops since a certain release. 
Greetings Kh

**Comment by radek8** [comment]
You will not see the storage history for a specific date, but for a specific Pull Request
For example, I want the state of the files before merging request 2869:

![image](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/46979052/ef7ea9d2-faad-4b14-952f-2e5cc8fcc29f)

![image](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/46979052/63caa77e-6230-4bf7-8635-84dff585c253)

And you have what you are looking for

![image](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/46979052/b2b3305e-6feb-4d43-be39-804b494a62bd)

**Reply by thisiskeithb** [reply]
The proper way to step through a repo's history when viewing on GitHub.com is to view the [Commits page](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/commits/master/): 

<img width="797" alt="image" src="https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/13375512/9943361b-e0cc-4904-9dc8-07ddf2c27fa0"><br /><br />

From there, you can view individual commits & browse the repo at that point in history by clicking on the `< >` button:

<img width="721" alt="image" src="https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/13375512/0d880b95-947d-4d88-bf73-2c95c08304a7">

**Comment by radek8** [comment]
It is better and more convenient to use the application:
GitHub Desktop
